it/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBusinessPostcodeControllerISpec.scala

```diff
@@ -57,12 +77,14 @@ class CaptureBusinessPostcodeControllerISpec extends ComponentSpecHelper with Ca
     }
 
     "return a BAD_REQUEST if the postcode is invalid" in {
+      stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
       lazy val result = post(s"/$testJourneyId/business-postcode")(
         "business_postcode" -> "111 111"
       )
 
       result.status mustBe BAD_REQUEST
-      }
+    }
```

Comment:
This test will need to be set up like the one you added above, to test the view error messages (see below for how to create them -- testCaptureBusinessPostcodeInvalidErrorViewTests) but with the invalid postcode in the request (instead of the empty request like above)